db/migrate/20200814143849_add_address_to_schools.rb

```diff
@@ -0,0 +1,10 @@
+class AddAddressToSchools < ActiveRecord::Migration[6.0]
+  def change
+    add_column :schools, :address_1, :string
+    add_column :schools, :address_2, :string
+    add_column :schools, :address_3, :string
```

Comment:
should any of these have `null: true` ?

Reply:
They get that by default? I would only normally add `null: false` if I wanted the 'NOT NULL' clause?

Reply:
no, I guess the default is fine 👍